README.md

```diff
@@ -154,15 +154,27 @@ piece of state must be named `routing`** (unless you provide a custom
 
 ### `UPDATE_PATH`
 
-An action type that you can listen for in your reducers.
+An action type that you can listen for in your reducers to be notified of
+route updates.
 
-### `updatePath(path, noRouterUpdate)`
+### `pushPath(path, state = {}, avoidRouterUpdate = false)`
```

Comment:
This does not match the api: https://github.com/jlongster/redux-simple-router/blob/3545a3e7fea0d8629180c774e16825d0d4ce547c/src/index.js#L23 (e.g. `state` does not have a default value)

I actually think you can just copy the line from the code.